cadc-storage-permissions/src/main/webapp/capabilities.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<vosi:capabilities
+    xmlns:vosi="http://www.ivoa.net/xml/VOSICapabilities/v1.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns:vs="http://www.ivoa.net/xml/VODataService/v1.1">
+
+  <capability standardID="vos://cadc.nrc.ca~vospace/CADC/std/inventory/#TBD-1.0">

```

Comment:
I chose vos://cadc.nrc.ca\~vospace/CADC/std/inventory#artifacts-1.0 for minoc, but it could perhaps be #files-1.0 instead.
This one could be vos://cadc.nrc.ca\~vospace/CADC/std/inventory#permissions-1.0

Reply:
permissions makes sense. We haven't named the project yet, hopefully tomorrow since I'm getting tired of typing cadc-storage-permissions, and was going to raise the standardID name then. Or just change it to permissions now.